akka-docs/rst/scala/distributed-data.rst

```diff
@@ -41,8 +41,10 @@ with a specific role. It communicates with other ``Replicator`` instances with t
 ``akka.cluster.ddata.DistributedData`` extension.
 
 Cluster members with status :ref:`WeaklyUp <weakly_up_scala>`, if that feature is enabled,
-will currently not participate in Distributed Data, but that is something that should be possible to
-add in a future release.
+will participate in Distributed Data. Note that it will not participate in any actions where the
```

Comment:
between those two sentences something like: "The data will be replicated to the `WeaklyUp` nodes with the background gossip protocol."